lib/widgets/theme.dart

```diff
@@ -182,6 +184,8 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       btnLabelAttLowIntDanger: const Color(0xffff8b7c),
       btnLabelAttMediumIntDanger: const Color(0xffff8b7c),
       composeBoxBg: const Color(0xff0f0f0f),
+      composeBoxBorderBar: Colors.white.withValues(alpha: 0.1),
```

Comment:
The Figma doesn't add a new design variable for the compose box border bar, it just changes the dark-theme value of an existing variable, `border-bar`. We should do the same.